php/test.php

```diff
@@ -142,16 +142,10 @@ private function checkExtensions()
 
         $errors             = array();
         $extensionsExpected = array(
-            'bcmath', 'exif', 'gd', 'iconv', 'intl', 'mbstring', 'pcntl', 'pdo_mysql',
-            'pdo_pgsql', 'redis', 'soap', 'sockets', 'xdebug', 'zip',
+            'apcu', 'bcmath', 'blackfire', 'exif', 'gd', 'iconv', 'intl', 'mbstring', 'memcached', 'pcntl', 'pdo_mysql',
+            'pdo_pgsql', 'pgsql', 'redis', 'soap', 'sockets', 'ssh2', 'xdebug', 'Zend OPcache', 'zip',
```

Comment:
`ssh2` must be omitted until support is ok.

BTW the build should be down. @Mallear something is broken, Travis should detect that `php test.php` returned a failure (through the returned status code), could you please have a look?

Reply:
Ok remove the check if ssh2 extension enabled